csharp-starter/functions/Hello.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Nitric.Sdk;
+using System;
+
+var serviceAddress = System.Environment.GetEnvironmentVariable("SERVICE_ADDRESS");
+
+Console.WriteLine(serviceAddress);
```

Comment:
Is this service address stuff needed? I'm assuming this is just leftover from some testing.